azure_arc_k8s_jumpstart/aro/arm_template/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
No ARM template so should be removed no?

Reply:
there may be an arm template in the future and that is the skeleton of the arm template.